AirCasting/SessionViews/ReconnectSessionCardView.swift

```diff
@@ -0,0 +1,101 @@
+// Created by Lunar on 21/10/2022.
+//
+
+import SwiftUI
+import AirCastingStyling
+import Resolver
+
+struct ReconnectSessionCardView: View {
+    let session: SessionEntity
+    @EnvironmentObject private var tabSelection: TabBarSelection
+    @EnvironmentObject var selectedSection: SelectedSection
+    @State private var alert: AlertInfo?
+    
+    var body: some View {
+        Group {
+            Spacer()
+            VStack(alignment: .leading, spacing: 5) {
+                header
+                content
+            }
+            .foregroundColor(.aircastingGray)
+            .padding()
+            .background(
+                Color.aircastingBackground
+                    .cardShadow()
+            )
+            .overlay(Rectangle().frame(width: nil, height: 4, alignment: .top).foregroundColor(Color.red), alignment: .top)
+        }
+    }
+    
+    var header: some View {
+        SessionHeaderView(
+            action: {},
+            isExpandButtonNeeded: false,
+            isMenuNeeded: false,
+            isCollapsed: .constant(false),
+            session: session)
+    }
+    
+    var content: some View {
+        VStack(spacing: 15) {
+            Text(Strings.ReconnectSessionCardView.heading)
+                .font(Fonts.moderateBoldHeading1)
+                .foregroundColor(.darkBlue)
+                .multilineTextAlignment(.center)
+            Text(Strings.ReconnectSessionCardView.description)
+                .font(Fonts.moderateRegularHeading3)
+                .multilineTextAlignment(.center)
+            reconnectionLabel
+            finishAndDontSyncButton
+                .padding()
+        }
+        .alert(item: $alert, content: { $0.makeAlert() })
+        .padding()
+    }
+    
+    var reconnectionLabel: some View {
```

Comment:
In my opinion this design should be rethinked because blue button suggest that it is tappable and some action will follow while it is not

Reply:
This is a design provided to me by Michael (it comes from Android) - and I don't think it is worth the effort to now change this design in both systems. What do you think?

![Screenshot_20221017-161911](https://user-images.githubusercontent.com/44276070/197533666-f37385bb-3771-44bf-bd00-1fbfbbe8bafb.png)

Reply:
I will ask Michael, never thought about this